framework/core/Project/Lang.pm

```diff
@@ -102,15 +122,58 @@ sub _layout2 {
     return {src=>$src, test=>$test};
 }
 
+##
+## Converts file encoding from iso-8859-1 to utf-8
+##
+sub convert_file_encoding {
+    @_ == 1 or die $ARG_ERROR;
+    my ($file_name) = @_;
+    if (-e $file_name){
+        rename($file_name, $file_name.".bak");
+        open(OUT, '>'.$file_name) or die $!;
+        my $converted_file = `iconv -f iso-8859-1 -t utf-8 $file_name.bak`;
+        print OUT $converted_file;
+        close(OUT);
+    }
+}
+
```

Comment:
Move to Utils and use it here and in Math (and possibly elsewhere)?

Reply:
moved to utils as part of Math pull request.

Reply:
I merged the Math PR, so you can remove this function and update the call sites.